src/models/oryp12/README.md

```diff
@@ -0,0 +1,70 @@
+# Oryx Pro (oryp12)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Oryx Pro](./img/oryp12.webp)
+
+The System76 Oryx Pro is a laptop with the following specifications:
+
+- CPU
+    - Supports Intel 14th Generation (Raptor Lake) CPUs
+        - [Intel Core i9-14900HX](https://ark.intel.com/content/www/us/en/ark/products/235995/intel-core-i9-processor-14900hx-36m-cache-up-to-5-80-ghz.html)
+- Chipset
+    - [Intel HM770](https://ark.intel.com/content/www/us/en/ark/products/232478/intel-hm770-chipset.html)
+- BIOS
+    - GigaDevice GD25B256EYIGR flash chip
+        - WSON-8 form factor
+    - Programmed with [System76 Open Firmware](https://github.com/system76/firmware-open)
+- EC
+    - ITE IT5570E
+    - Programmed with [System76 EC](https://github.com/system76/ec)
+- Graphics
+    - Dedicated GPU options:
+        - NVIDIA GeForce RTX 4070
+            - TGP: 115W (140W boost)
+        - NVIDIA GeForce RTX 4060
+            - TGP: 115W (140W boost)
+        - NVIDIA GeForce RTX 4050
+            - TGP: 115W (140W boost)
+    - Integrated GPU:
+        - Intel Iris Xe Graphics
+    - eDP display options:
+        - 16" 1920x1200@165Hz LCD
+            - LCD panel: BOE NV160WUM-NX2 (or equivalent)
+                - Brightness: 500 nits (cd/m²)
+                - sRGB color coverage: 100%
+    - External video output:
+        - 1x HDMI 2.1
+        - 1x Mini DisplayPort 1.4
+        - 1x DisplayPort 1.4 over USB-C
+- Memory
+    - Up to 96GB (2x48GB) dual-channel DDR5 SO-DIMMs @ 5600 MHz
```

Comment:
This should be 5200Mhz as the firmware does not support anything above that.

Reply:
That is not correct. Oryp11 was limited to 5200MHz because its CPU was a 13900H. Oryp12 uses a 14900HX, which supports 5600MHz.